juanevillam/xtrovert
# Configure Font and Tailwind CSS Setup

## Overview  
This PR configures the Poppins font and completes the Tailwind CSS setup. It also reorganizes the `globals.css` file for better project structure.

---

## Key Changes  

### **Features**  
- Configured the Poppins font for consistent typography across the application.  
- Completed Tailwind CSS configuration to ensure all utility classes are available.  

### **Enhancements**  
- Moved `globals.css` to `src/assets/ui/styles/` for improved organization and maintainability.  

---

## Reasons for Changes  
Setting up the Poppins font and properly configuring Tailwind CSS ensures a cohesive design system and consistent styling throughout the application. Reorganizing the `globals.css` file enhances the overall structure and readability of the project.

---

## How to Test  
1. Run the application to ensure it builds without errors.  
2. Verify that the Poppins font is correctly applied across all components.  
3. Test various Tailwind CSS classes to confirm they are working as expected.  
4. Ensure that moving `globals.css` has not broken any global styles.
